torch/testing/_internal/torchbind_impls.py

```diff
@@ -28,5 +28,15 @@ def fake_queue_push(tq, x):
         return tq.push(x)
 
     @register_if_not("_TorchScriptTesting::queue_size")
-    def fake_queue_size(tq, x):
+    def fake_queue_size(tq):
         return tq.size()
+
+    torch.ops._TorchScriptTesting.queue_pop.default.fallthrough(
```

Comment:
We explicitly fallthrough AutogradCPU key.

Reply:
We should put this into the fallthroughs that we make in TorchBindOp's ctor